streamz/core.py

```diff
@@ -481,9 +483,12 @@ def __init__(self, upstream, func, *args, **kwargs):
         Stream.__init__(self, upstream, stream_name=stream_name)
 
     def update(self, x, who=None):
-        result = self.func(x, *self.args, **self.kwargs)
-
-        return self._emit(result)
+        try:
+            result = self.func(x, *self.args, **self.kwargs)
+        except Exception as e:
+            logger.exception(e)
```

Comment:
Do we want an option to raise?

Reply:
Eventually we probably want to percolate exceptions as we do data or futures.  I don't know what defaults should be in the meantime.

Reply:
If we should raise I'm fine with that

Reply:
We now raise